lib/fluent/plugin/output.rb

```diff
@@ -1224,6 +1226,7 @@ def try_flush
             dump_chunk_id = dump_unique_id_hex(chunk_id)
             log.trace "adding write count", instance: self.object_id
             @write_count_metrics.inc
+            @write_secondary_count_metrics.inc if using_secondary
```

Comment:
Don't we also need to count it in the case of `delayed_commit`?

https://github.com/fluent/fluentd/blob/0b72a77d23a667c2c5c648e457e16fc8835c2134/lib/fluent/plugin/output.rb#L1212-L1215

Reply:
Ah, thanks.

Reply:
Added https://github.com/fluent/fluentd/pull/4971/commits/70d80b5cc861fffa577a0603205b5015a3e23094